src/coreclr/jit/layout.h

```diff
@@ -195,6 +195,8 @@ class ClassLayout
         }
     }
 
+    static constexpr ClassLayout* NO_LAYOUT = nullptr;
```

Comment:
Deleted.

To me it just looked nicer to have `SetLayout(ClassLayout::NO_LAYOUT)` instead of `SetLayout(nullptr)`, but I see how it can be non-obvious what the value is.